brave/brave-core
# Remove DDG search toggle from private tabs

Resolves https://github.com/brave/brave-browser/issues/18486

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Clean profile
- Set `brave.use_alternate_private_search_engine` preference to `false`
- Launch browser
- Open Private Tab
- Verify that Private Tab shows new design and no DDG toggle
- Clean profile
- Set `brave.use_alternate_private_search_engine` preference to `true`
- Launch browser
- Open Private Tab
- Verify that Private Tab shows new design and DDG toggle
- Verify that toggle works as expected
